Article/Article.js

```diff
@@ -3,18 +3,20 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'expand';
     // Set a click handler on the expandButton reference, calling the expandArticle method.
+    this.expandButton.addEventListener('click', this.expandArticle.bind(this)); //Using bind so the expandArticle method can use this reference
```

Comment:
I would practice the ES5 way here without the bind. this.expandArticle(). This looks cleaner. I'd practice ES6 going forward just because it is going to only help you going forward.